performance/buildSrc/src/main/kotlin/MPPTools.kt

```diff
@@ -84,8 +84,21 @@ fun getFileSize(filePath: String): Long? {
     return if (file.exists()) file.length() else null
 }
 
-fun getCodeSizeBenchmark(programName: String, filePath: String): BenchmarkResult {
-    val codeSize = getFileSize(filePath)
+fun getFolderSize(folderPath: String): Long? {
```

Comment:
Btw, frameworks on macOS have symlinks inside. Does this function support symlinks properly?